### PR TITLE
【PaddlePaddle Hackathon 4 】add silu for paddle

### PR DESCRIPTION
**Pull request by taixiurong**
### Details:
add silu for paddle

### Reference:
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Silu_cn.html#silu

### Test
35cff261e003becbda310c850402ce05

### CODE REVIEW

**Comment by taixiurong** [comment]
@ceciliapeng2011  please review this silu

File: src/frontends/paddle/src/op/silu.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs silu(const NodeContext& node) {
+    auto x = node.get_input("X");
+    auto neg_x = std::make_shared<default_opset::Negative>(x);
+    auto exp_neg_x = std::make_shared<default_opset::Exp>(neg_x);
+    auto const_one = default_opset::Constant::create<float>(ov::element::f32, {1}, {1});
+    auto denominator = std::make_shared<default_opset::Add>(const_one, exp_neg_x);
+    return node.default_single_output_mapping({std::make_shared<default_opset::Divide>(x, denominator)}, {"Out"});
```
Comment (mvafin) [inline src/frontends/paddle/src/op/silu.cpp:18]: Please use [Swish](https://docs.openvino.ai/latest/openvino_docs_ops_activation_Swish_4.html) operation from openvino opset.

**Review by mvafin — CHANGES_REQUESTED** [review]

**Comment by akladiev** [comment]
This PR will be closed in a week because of 2 weeks of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**